src/GrammarParser.php

```diff
@@ -85,13 +85,11 @@ public function buildGrammar($grammarStr, $options = [])
 
         $grammarBranches = $parsedGrammar->findAll('grammarBranch');
 
-        $defaultBranchType = empty($options['defaultBranchType']) ? \ParserGenerator\GrammarNode\BranchFactory::FULL : $options['defaultBranchType'];
```

Comment:
Temporary var not necessary anymore, as we can directly read the value; an `FULL` is the default anyways.